generate_report.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
This is a lot of CI files at the project root... I understand the .yml files need to be there for the CI services to pick up, but can we move the other utilities into a CI folder? (Probably under //CommandLine/ or //CommandLine/testing/)